gnome-shell/src/gnome-shell-sass/widgets/_search-results.scss

```diff
@@ -22,11 +22,11 @@
 
 // content
 .search-section-content {
-  background-color: rgba($osd_bg_color, 0.1);
+  background-color: rgba($osd_bg_color, 0.2);
   border-radius: $modal_radius+3;
   border: 1px solid $osd_outer_borders_color;
   box-shadow: 0 2px 4px 0 $shadow_color;
-  text-shadow: 0 1px if($variant == 'light', rgba(255,255,255,0.2), rgba(0,0,0,0.2));
+  //text-shadow: 0 1px if($variant == 'light', rgba(255,255,255,0.2), rgba(0,0,0,0.2));
```

Comment:
Should this comment be removed?

Reply:
I think this will reduce our delta with upstream. However, if we prefer I can remove it too.